### PR TITLE
Index country on ISO code

### PR DESCRIPTION
**Pull request by umpirsky**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| License       | MIT

### CODE REVIEW

File: src/Sylius/Bundle/AddressingBundle/Resources/config/doctrine/model/Country.orm.xml [hunk 1]
```diff
@@ -14,11 +14,14 @@
...
     <mapped-superclass name="Sylius\Component\Addressing\Model\Country" table="sylius_country">
+        <indexes>
+            <index columns="iso_name" />
+        </indexes>
         <id name="id" column="id" type="integer">
             <generator strategy="AUTO" />
         </id>
 
-        <field name="isoName" column="iso_name" type="string" />
+        <field name="isoName" column="iso_name" type="string" length="2" />
```
Comment (kayue) [inline src/Sylius/Bundle/AddressingBundle/Resources/config/doctrine/model/Country.orm.xml]: @umpirsky Maybe we shouldn't limit the length?

Both "US" and "USA" should be allowed.

Comment (umpirsky) [inline src/Sylius/Bundle/AddressingBundle/Resources/config/doctrine/model/Country.orm.xml]: @kayue Well, there are [ISO 3166-1 alpha-2](http://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) and [ISO 3166-1 alpha-3](http://en.wikipedia.org/wiki/ISO_3166-1_alpha-3). In fixtures we used ISO 3166-1 alpha-2. Maybe we can update length to 3 and support both.

Thoughts @pjedrzejewski?

Comment (pjedrzejewski) [inline src/Sylius/Bundle/AddressingBundle/Resources/config/doctrine/model/Country.orm.xml]: :+1:

Comment (umpirsky) [inline src/Sylius/Bundle/AddressingBundle/Resources/config/doctrine/model/Country.orm.xml]: @pjedrzejewski Fixed.

**Pull request merged**